packages/bbui/src/Form/Core/DateRangePicker.svelte

```diff
@@ -2,13 +2,6 @@
   import CoreDatePicker from "./DatePicker/DatePicker.svelte"
   import Icon from "../../Icon/Icon.svelte"
 
-  export let value = null
-  export let disabled = false
-  export let readonly = false
-  export let error = null
-  export let appendTo = undefined
-  export let ignoreTimezones = false
```

Comment:
Unused?

Reply:
Yes. And typescript complained about that 😄

Reply:
I've actually removed this file altogether in my branch. BBUI is consumed by frontend-core so I saw the same issues, and this file is actually entirely unused. You can leave it in though - I can deal with the small conflict.

Reply:
I had the feeling that it was not used at all. I just did not want to remove it just in case some old app was using it somehow 🪄